From d77594a0fb9b56aff92cd555ae7b2404de61d137 Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Sat, 11 May 2019 18:39:50 +0200
Subject: [PATCH 282/345] MIPS: Pass -fno-asynchronous-unwind-tables to
 compiler.

Some toolchains default to -fasynchronous-unwind-tables.  Since the
resulting .eh_frame sections are omitted by the linker scripts, we are
left with no unwind information for analyzing core dumps.

Pass -fno-asynchronous-unwind-tables so the .debug_frame is generated
instead.

Signed-off-by: David Daney <david.daney@cavium.com>
---
 arch/mips/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/Makefile b/arch/mips/Makefile
index 068cabefc5e2..caa7092c365d 100644
--- a/arch/mips/Makefile
+++ b/arch/mips/Makefile
@@ -91,7 +91,7 @@ all-$(CONFIG_SYS_SUPPORTS_ZBOOT)+= vmlinuz
 # crossformat linking we rely on the elf2ecoff tool for format conversion.
 #
 cflags-y			+= -G 0 -mno-abicalls -fno-pic -pipe -mno-branch-likely
-cflags-y			+= -msoft-float
+cflags-y			+= -msoft-float -fno-asynchronous-unwind-tables
 LDFLAGS_vmlinux			+= -G 0 -static -n -nostdlib
 
 ifndef CONFIG_MAPPED_KERNEL
-- 
2.25.1

